src/extension.ts

```diff
@@ -192,4 +203,13 @@ function runSelectedAsQuery(): void {
   }
 }
 
+function dryRun(): void {
+  try {
+    let queryText = getQueryText(vscode.window.activeTextEditor);
+    query(queryText, true);
```

Comment:
`query` calls `writeResults` - but that function only writes the result table out, not the job statistics. For a dry run, you very likely want to see bytes scanned, to determine whether your query was not only syntactically correct, but also hit the right partitions/clusters/isn't going to bill you too much.

Reply:
Done, added `function writeDryRunSummary(jobId: string, numBytesProcessed: string)`